pkg/apiserver/api_installer.go

```diff
@@ -104,9 +104,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	admit := a.group.Admit
 	context := a.group.Context
 
-	serverGroupVersion := a.group.GroupVersion
+	externalGroupVersion := a.group.GroupVersion
 	if a.group.ServerGroupVersion != nil {
-		serverGroupVersion = *a.group.ServerGroupVersion
+		externalGroupVersion = *a.group.ServerGroupVersion
```

Comment:
@liggitt, yes, it is an "external" version, I agree with this part, but I think naming the variable to `ExternalGroupVersion` is confusing, because we usually use this term to refer to the external version of `a.group`, i.e., https://github.com/kubernetes/kubernetes/pull/18379/files#diff-fdfac9813673596faeda07fdce9207e2R84.
But the version here, as you pointed out, is used to get the versioned options, and it is always set to "v1", so I think naming it `ExternalGroupVersion` is confusing.